Dockerfile

```diff
@@ -0,0 +1,14 @@
+FROM python:3.8-slim
+
+ARG NAME
+ARG SPACY_MODEL
+WORKDIR /usr/bin/${NAME}
+
+COPY ./${NAME} /usr/bin/${NAME}
+
+RUN pip install pipenv
+RUN pipenv sync
```

Comment:
Kept pipenv instantiation from the lock file, although not the best practice. @omri374 stated that it solved issues in the past.

Reply:
If we decide to go this way, then I would use the same way which we had in V1: https://github.com/microsoft/presidio/blob/4b3f637a6337fcdf54a07f5d1e78559dd68945a1/Dockerfile.python.deps#L26

Let's discuss offline.

Reply:
If we decide to go this way, then I would use the same way which we had in V1: https://github.com/microsoft/presidio/blob/4b3f637a6337fcdf54a07f5d1e78559dd68945a1/Dockerfile.python.deps#L26

Let's discuss offline.